registry.k8s.io/images/charts/images.yaml

```diff
@@ -7,3 +7,6 @@
 - name: jobset
   dmap:
     "sha256:e1f25f53ba4312f99bb60b92668d98104c549067d86756d30cbf1cf18f2b47f7": ["v0.8.0"]
+- name: lws
+  dmap:
```

Comment:
/hold

please don’t add this here. We are asking maintainers to put their charts with the images.

the policy changed and I am trying to document it but that hasn’t been approved yet.

Reply:
https://github.com/kubernetes/k8s.io/pull/7915

Reply:
This makes more sense to me.

Reply:
What the structure looks like? Do we have an example?

Reply:
@kannon92

Reply:
leave one comment here: https://github.com/kubernetes/k8s.io/pull/7915#discussion_r2020227141

Reply:
https://github.com/kubernetes/k8s.io/blob/main/registry.k8s.io/images/k8s-staging-kueue/images.yaml#L1

Reply:
Thanks @kannon92